### ISSUE TITLE
Not able to access website in certain scenarios

### ISSUE THREAD

**Issue by werunom** [bitwarden/web#195]
I have been unable to access the bitwarden website in certain scenarios. Wanted to discuss them in case there are certain settings that I need to change.

Scenario#1 - Accessing bitwarden behind a proxy firewall
My working place has wifi access, but through a proxy firewall. So, when i try to access the website `https://bitwarden.com/` (so that I can access the web vault), I get the below message in `firefox` and `Manjaro linux`

![screenshot_20180328_102043](https://user-images.githubusercontent.com/24664855/38009494-3b47d4d8-3272-11e8-8d20-ab1c891d69a5.png)

Note - its the same case if I try `https://vault.bitwarden.com/`

Scenario#2
When I tried to access the website from my friend's laptop using `chrome` running in `win xp`, I got the similar message about HSTS. 

The reason I thought of bringing this up is for the following reasons -

1. I had only encountered `scenario#1` and the way I reasoned it out is by considering that the proxy at work place is getting confused with the strict security protocol of bitwarden. And really, this was not a concern since I have bitwarden desktop app to use. (BTW, the desktop app also does not sync when I am behind proxy.)
But in `scenario#2`, when I was using my friend's laptop and I needed to access my mails, the above issue was a major setback. Good that I had my phone, but you can imagine the struggle I had to copy the 30 characters long password.

2. But in the both the scenarios, I was able to access lastpass website. Let me confess: I do not want to compare and contrast; I only wanted to check whether all the password-management websites have similar "security concerns" and behave similarly in both the scenarios. Nor i am saying that lastpass is better, etc. It might be the case that bitwarden has another layer of security (or something like that).

Basically, I have no clue what this situation is and why is it happening. So, thought of putting it here to know more about this and in case if its a point of concern. 

But as an end user, I still want to know whether there is a work around to overcome this? The whole point of shifting from keypassxc to bitwarden was to be able to access the passwords in situations like `scenario#2`.

**Comment by kspearrin** [comment]
Very weird. The only thing that I can think might be the causes are:

1. Your networks are using MITM configurations to alter certificates, causing these types of issues. Though I would think you would see them present on other websites if so.
2. For some reason these machins have corrupt HSTS settings. You could try resetting them for the bitwarden.com domains. What if you tried another web browser?

**Comment by werunom** [comment]
The web-vault opens fine in my laptop when I open it in other networks - like my home and other places.  Only in my institute's wifi network, it does not open. But, as I mentioned, lastpass website opens. (And today i tried creating an account there and everything goes fine.) So, would not that invalidate point (2)? 

And bit confused with you what you mean by "machines" in point (2). Are you referring to my laptop or the network wifi systems in my working place? 

Even if what you are saying is correct, that just validates `scenario#1`; but `scenario#2` is unexplained. As I mentioned, only in my friend's laptop, the website does not open. (It might also be the case that he is using a very old version of Chrome.)

In any case, below are the scenarios that I can test if you can tell me some way to debug:
1. I can ask my friend to try opening the website in someother laptop or mobile. The very laptop in which I tried is not present (long story short - someone else taken it).
2. I can try opening the website from my institute's wifi. 

In both the cases, if you tell me what to check, I can investigate and get back to you.

**Comment by kspearrin** [comment]
No, since your browser could still have corrupted cached values for bitwarden HSTS. Trying another web browser would verify this.

Machine = device. Your laptop/browser.

Just try another web browser in each scenario and see if that changes anything.

**Comment by werunom** [comment]
Just updated my comment. I have already tried opening bitwarden in another desktop connected to the office network. In that also I was unable to access. 

But tomorrow, I will try accessing it from other laptops and different browser from my laptop.

**Comment by kspearrin** [comment]
Could you also try accessing https://api.bitwarden.com/now and see if that shows the current time or any errors?

**Comment by werunom** [comment]
Tried couple of things.

**Scenario#1**

1. Tried accessing https://bitwarden.com and https://api.bitwarden.com/now  using `chrome` in my laptop when it is connected to institute wifi. But now, instead of the HSTS issue, the chrome shows the right screen - the firewall blocking the websites. 

![screenshot_20180329_110338](https://user-images.githubusercontent.com/24664855/38072101-84ba517e-3342-11e8-8779-c824acb7d695.png)

But this is not unexpected. As i had thought, the firewall is blocking the access. 

**Scenario#2**
I called up my friend and asked her to access the website in her laptop and mobile (connected to the same old wifi in which I had faced the problem earlier). There, everything works as intended. But as mentioned, the laptop in which I saw the HSTS message is not there to retest the scenario.

Since this issue finally boils down to only one scenario (using a specific laptop in scenario#2) that cannot be retested, I think there is no point of concern...unless someone else is also facing this. 

Please feel free to close this issue.
